nservicebus/databus.md

```diff
@@ -0,0 +1,69 @@
+---
+title: DataBus feature
+summary: DataBus feature
+tags:
+- DataBus
+- Attachments
+---
+
+## What is DataBus feature for?
+
+Messages are intended to be small. Some scenarios require to send large binary data along with a message. For this purpose, NServiceBus has a DataBus feature to allow to overcome limitations of message size.
+
+## How it works?
+
+`DataBus` is taking an approach of storing large payload and storing it in at a location both sending and receiving parties can access. Message is sent with a reference to the location, and upon processing payload is brought, allowing receiving part access message along with payload. In case location is not available upon sending, message will fail send operation. Receive operation will fail as well when payload location is not available and will cause standard NServiceBus behaviour, causing retries and eventually going into error queue.
+
+## How to enable DataBus
+
+NServiceBus supports 2 DataBus implementations:
+
+* `FileShareDataBus`
+* `AzureDataBus`
+
+To enable DataBus, NServiceBus needs to be configured. For file share based DataBus:
+<!-- import FileShareDataBus -->
+
+For Azure (storage blobs) based DataBus:
+<!-- import AzureDataBus -->
+
+Note: The `AzureDataBus` implementation is part of the Azure transport package.

```

Comment:
?d add that the AzureDataBus can be used even if the transport is not Azure based.